packages/commonwealth/server/routes/bulkOffchain.ts

```diff
@@ -47,6 +49,7 @@ const bulkOffchain = async (models: DB, req: Request, res: Response) => {
         RoleInstanceWithPermission[],
         unknown,
         ThreadInstance[],
+        unknown,
```

Comment:
Can we use a proper type here rather than unknown?

Reply:
Fixed [here](https://github.com/hicommonwealth/commonwealth/pull/3534/commits/e5dce18ad085f0d22b0028c4e3c95fcb29a7b6d9)